src/app/app.js

```diff
@@ -0,0 +1,21 @@
+angular.module('esnApp', [
+  'esn.calendar',
+  'linagora.esn.resource',
+  'ui.router',
+  'esn.session',
+  'esn.websocket',
+  'esn.configuration',
+  'esn.i18n',
+  'xeditable',
+  'openpaas-logo'
+]);
+
+require('esn-frontend-common-libs/src/frontend/js/modules/session');
+require('esn-frontend-common-libs/src/frontend/js/modules/websocket');
+require('esn-frontend-common-libs/src/frontend/js/modules/config/config.module');
```

Comment:
This shouldn't work since config.module is not a file? Should be `config.module.js`

Reply:
If it is really the webpack that I know, its module resolution doesn't require explicitly specifiying the file extensions. So both `require('something')` and `require('something.js')` should work by default.

Reply:
Huy is correct on that.